backend/src/components/application.js

```diff
@@ -598,6 +606,45 @@ async function updateStatusForApplicationComponents(req, res) {
   }
 }
 
+async function populateSummaryDataForFacility(facility) {
+  try {
+    const childCareLicenses = await getLicenseCategoriesByFacilityId(facility.facilityId);
+    facility.childCareLicenses = Array.from(childCareLicenses.values());
+  } catch (e) {
+    log.warn('populateSummaryDataForFacility unable to find License Categories', e);
+  }
+
+  // check for opt out - no need for more calls if opt-out
+  if (facility.ccfri?.ccfriId && facility.ccfri?.ccfriOptInStatus == 1) {
+    const { ccfri } = facility;
+    const facilityChildcareTypes = await getFacilityChildCareTypesByCcfriId(ccfri.ccfriId);
+    facility.ccfri.childCareTypes = facilityChildcareTypes.childCareTypes;
+    facility.ccfri.dates = facilityChildcareTypes.dates;
+
+    // load up the previous ccfri app if it exists, so we can check that we are not missing any child care fee
+    // categories from the last year.
+    if (ccfri.previousCcfriId) {
+      const previousCcfriId = await getFacilityChildCareTypesByCcfriId(ccfri.previousCcfriId);
+      facility.ccfri.prevYearCcfriApp = previousCcfriId;
+    }
+
+    if (ccfri?.hasRfi || ccfri?.unlockRfi) {
```

Comment:
We are null checking ccfri here but not on the above lines. Is that something that needs to be added in?

Reply:
Probably not because `ccfri.previousCcfriId` is not contingent on unlocking maybe?